pkgs/tools/networking/subnetcalc/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchurl, cmake, ninja }:
+
+stdenv.mkDerivation rec {
+  pname = "subnetcalc";
+  version = "2.4.19";
+
+  src = fetchurl {
+    url = "https://www.uni-due.de/~be0001/subnetcalc/download/subnetcalc-${version}.tar.xz";
+    sha256 = "sha256-E/NavAeCx0U9oiYCEo65P6ZFA52SzVqzxSiunmAyzWc=";
+  };
```

```suggestion
{ lib, stdenv, fetchFromGitHub, cmake, ninja }:

stdenv.mkDerivation rec {
  pname = "subnetcalc";
  version = "2.4.19";

  src = fetchFromGitHub {
    owner = "dreibh";
    repo = pname;
    rev = "${pname}-${version}";
    sha256 = "sha256-70OHHcdCsZBa95KQEEYhs6LfpjPg+ULOB+A+VrQVltU=";
  };
```

Suggestion: Not sure what the general preference is, but maybe getting tarballs from github is better for preservation in the long term?

---

Done!

---

Mostly irrelevant.

Many projects use github merely to save the code, the targz links are a bonus. However, sometimes the github sources differ a bit, requiring extra tools like `autoreconfHook`.